execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -826,6 +836,9 @@ where
 
         let (_, handle_payment_hash) = self.store_contract(access_key, named_keys, entry_points);
 
+        self.store_system_contract(HANDLE_PAYMENT, handle_payment_hash)
+            .expect("should install handle payment");
+
```

Comment:
To be consistent with other `store_system_contract` calls:

```suggestion
        self.store_system_contract(HANDLE_PAYMENT, handle_payment_hash)?;
```

Reply:
Applied suggestion